src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.cs

```diff
@@ -351,20 +389,17 @@ public static Vector4 In(Vector4 destination, Vector4 source)
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     public static Vector256<float> In(Vector256<float> destination, Vector256<float> source)
     {
-        const int blendAlphaControl = 0b_10_00_10_00;
-        const int shuffleAlphaControl = 0b_11_11_11_11;
-
         // calculate alpha
-        Vector256<float> sW = Avx.Shuffle(source, source, shuffleAlphaControl);
-        Vector256<float> dW = Avx.Shuffle(destination, destination, shuffleAlphaControl);
+        Vector256<float> sW = Avx.Shuffle(source, source, ShuffleAlphaControl);
+        Vector256<float> dW = Avx.Shuffle(destination, destination, ShuffleAlphaControl);
         Vector256<float> alpha = Avx.Multiply(sW, dW);
```

Comment:
You should be able to change this to:
```csharp
Vector256<float> alpha = Avx.Permute(Avx.Multiply(source, destination), ShuffleAlphaControl);
```

That will do a single shuffle/permute, rather than two of them. In general, if you can do ops and shuffle less after its "better" because shuffle ports are typically limited.

In this case its safe since all the operations are `element-wise`, so even though you're multiplying `source[i] * destination[i]`, the ones for `i = 0, 1, 2` don't matter and just get ignored in the subsequent permute.

It's possible there are similar opportunities in the shuffle ops in the methods above here, but they are "more complex" than this one so I didn't do any in depth analysis to look for the same simplification/optimizations in them